documentation/src/components/layout.ts

```diff
@@ -29,13 +36,61 @@ export class LayoutElement extends RouteComponent {
     render() {
         return html`
             <sp-theme color="light" scale="medium" id="app">
+                <header>
+                    <sp-action-button
+                        quiet
+                        aria-label="Open Navigation"
+                        @click=${this.toggleNav}
+                    >
+                        <svg
+                            slot="icon"
+                            viewBox="0 0 36 36"
+                            focusable="false"
+                            aria-hidden="true"
+                            role="img"
+                            width="18"
+                            height="18"
+                            fill="currentColor"
+                        >
+                            <rect
+                                height="4"
+                                rx="1"
+                                ry="1"
+                                width="28"
+                                x="4"
+                                y="16"
+                            ></rect>
+                            <rect
+                                height="4"
+                                rx="1"
+                                ry="1"
+                                width="28"
+                                x="4"
+                                y="6"
+                            ></rect>
+                            <rect
+                                height="4"
+                                rx="1"
+                                ry="1"
+                                width="28"
+                                x="4"
+                                y="26"
+                            ></rect>
+                        </svg>
+                    </sp-action-button>
+                </header>
                 <div id="body">
-                    <docs-side-nav id="side-nav"></docs-side-nav>
-                    <div id="layout-content">
+                    <docs-side-nav
+                        id="side-nav"
+                        ?inert=${!this.open}
```

Comment:
It _may_ be that all browsers need a polyfill for `inert` I didn't fully run this down, as I wasn't sure it's needed explicitly in the PR, but can follow up or remove if we'd like.